application/common/components/adapters/SagePeopleIdStore.php

```diff
@@ -259,10 +266,7 @@ protected function getFromIdStore($whereClause): array
      */
     public function getAllActiveUsers()
     {
-        $allActiveUsers = $this->getFromIdStore(
-            "WHERE fHCM2__Employment_Status__c='Active' AND fHCM2__User__c!=null AND "
-            . "fHCM2__Department__c!='a0H1U000001NKk4UAG'"
-        );
+        $allActiveUsers = $this->getFromIdStore('WHERE ' . $this->queryConditions);
```

Comment:
I always cringe a little when we're manually assembling SQL. Is there a way this could modified to use Yii's parameterized SQL query features (e.g. passing in the SQL with placeholders, and seperately providing the values for those placeholders)? This is slightly outside of the scope of this PR, but seems relevant.

Reply:
Good idea. I will add a Github issue.